tests/stake-pools/e2e/steps/stake-pools.js

```diff
@@ -153,7 +159,7 @@ Then(/^Stake pool "([^"]*)" tooltip shows correct data$/, async function (positi
   expect(secondStakePool.description).to.equal(stakePoolDescription);
 });
 
-When(/^I click on "Delegate to this pool"/, function () {
+When(/^I click on "([^"]*)"$/, function () {
```

Comment:
@DeeJayElly my bad, please revert this to `I click on "Delegate to this pool` because you pass param but param not used here. It is button label and that is legit